packages/fast-form-generator-react/README.md

```diff
@@ -198,8 +198,8 @@ import Form from "@microsoft/fast-form-generator-react";
                 title: "Formatting",
                 weight: 40,
                 properties: [
-                    { weight: 9, propertyName: "alignVertical" },
-                    { weight: 7, propertyName: "alignHorizontal" }
+                    { weight: 9, propertyName: "objectAlign" },
+                    { weight: 7, propertyName: "textAlign" }
```

Comment:
Is this really guaranteed to always be text? I believe we originally created it this way because we wanted to be able to align anything horizontally (say an image)

`textAlignSpacingForImage` feels very strange for me.

What was really wrong with `alignHorizontal`?

Reply:
image would likely be object alignment from my understanding.

Reply:
But `objectAlign` only has `[top, center, bottom]` in this PR  i.e. vertical alignment

Reply:
@eljefe223 who opened the issue. @janechu as a code owner. This was previously alignVertical; @nalogiudice is asking regarding the rename from alignVertical to what is now just "align". I have no thoughts, just looking at landing this for 4.0